600.java

```diff
@@ -56,5 +56,40 @@
 2. Consider simulating the deque as a way to efficiently determine which operations were performed on it. For example, as above, if the first element of the input is a 4, simulate a deque having the elements 1 through 4 pushed into it. Since you do not know whether each element was pushed to the front or the back, try pushing it on both sides and figuring out which side is correct later in the simulation.
```

Comment:
Next time, each line should no more than 80 characters. Easier for me to read the problem.